### ISSUE TITLE
PHP 7.4 support

### ISSUE THREAD

**Issue by xini** [bummzack/swiftmailer-emogrifyplugin#1]
Can you please update this to use pelago/emogrifier ^3.0 to support PHP 7.4?
Thanks.

**Comment by lozcalver** [comment]
PR: https://github.com/bummzack/swiftmailer-emogrifyplugin/pull/2. If anyone needs this in the meantime, I’m running it from my fork like this:

```json
{
    "require": {
        "bummzack/silverstripe-emogrify": "^0",
        "bummzack/swiftmailer-emogrifyplugin": "dev-master#90bc616 as 0.1.1"
    },
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/kinglozzer/swiftmailer-emogrifyplugin"
        }
    ],
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

**Comment by bummzack** [comment]
Released as version `0.2`
